interop/xds/server/server.go

```diff
@@ -78,8 +87,48 @@ func (s *testServiceImpl) EmptyCall(ctx context.Context, _ *testpb.Empty) (*test
 }
 
 func (s *testServiceImpl) UnaryCall(ctx context.Context, in *testpb.SimpleRequest) (*testpb.SimpleResponse, error) {
+	response := &testpb.SimpleResponse{ServerId: s.serverID}
+
+	for _, val := range getRPCBehaviorMetadata(ctx) {
+		if st := getStatusForRPCBehaviorMetadata(val); st != nil {
```

Comment:
Are you not planning to implement the complete spec here? https://github.com/grpc/grpc/blob/master/doc/xds-test-descriptions.md#server

```
In addition, when handling requests, if the initial request metadata contains the rpc-behavior key, it should modify its handling of the request as follows:

If the value matches sleep-<int>, the server should wait the specified number of seconds before resuming behavior matching and RPC processing.
If the value matches keep-open, the server should never respond to the request and behavior matching ends.
If the value matches error-code-<int>, the server should respond with the specified status code and behavior matching ends.
If the value matches success-on-retry-attempt-<int>, and the value of the grpc-previous-rpc-attempts metadata field is equal to the specified number, the normal RPC processing should resume and behavior matching ends.
A value can have a prefix hostname=<string> followed by a space. In that case, the rest of the value should only be applied if the specified hostname matches the server's hostname.
The rpc-behavior header value can have multiple options separated by commas. In that case, the value should be split by commas and the options should be applied in the order specified. If a request has multiple rpc-behavior metadata values, each one should be processed that way in order.
```

Reply:
Yeah, if you want please go ahead and implement the full spec. If not, then once this is merged I'll leave the issue open with the full bullets, and perhaps I can pick it up in the future. I guess I linked the C++ PR in the issue that only adds error code logic. Right now with this change, I don't think we can specify our interop tests use Go Server Side.